public/app/plugins/datasource/grafana-azure-monitor-datasource/components/MetricsQueryEditor/DimensionFields.tsx

```diff
@@ -44,7 +44,11 @@ const useDimensionLabels = (data: PanelData | undefined, query: AzureMonitorQuer
     }
     setDimensionLabels((prevLabels) => {
       const newLabels: DimensionLabels = {};
-      for (const label of Object.keys(labelsObj)) {
+      const currentLabels = Object.keys(labelsObj);
+      if (currentLabels.length === 0) {
+        return prevLabels;
+      }
```

Comment:
this change seems unrelated to the PR?

Reply:
This was a bug fix for an issue I noticed with the labels. As it was so short I figured I'd just add it here.

Reply:
that's fine for small patches :+1: What I usually do with these is either add a comment to my own PR review to highlight it or mention it in the PR description so the reviewer knows (and it's clear is not some leftover from the development)